### PR TITLE
DevDoc proddeploy updates

### PR DESCRIPTION
**Pull request by devjohnsonAMI**
Jira Ticket - https://cm-jira.usa.gov/browse/LG-7589

DevDoc updates per https://docs.google.com/document/d/1JrseKerp-FF_uiGElew9z-ltT_l68dgmSFU7so8tQjE/edit#

### CODE REVIEW

File: _pages/production.md [hunk 1]
```diff
@@ -2,81 +2,153 @@
...
+
+**When you have the components required, follow these steps to create your production configuration app:**
 
-## Confirm IAA
+1.  [Create a new app on the Partner dashboard](https://dashboard.int.identitysandbox.gov/). Select "Apps" from the top right menu, then select the "Create a new test app" button.  
 
-You must have a signed IAA (Inter-Agency Agreement) with Login.gov **with your integration explicitly listed in it** in order to deploy to production. You will need to provide the IAA number this application will be billed under. The IAA number format will include `GTC-Order-Mod` (e.g. `LGABCFY210001-0001-0000`), where GTC stands for General Terms & Conditions. You may also hear these referred to as forms 7600A and 7600B.
 
+2.  Select "YES" for production configuration - this configuration is for a production app.
+=======
```
Comment (zachmargolis) [inline _pages/production.md:79]: is this `=====` in here as part of a merge commit?
```suggestion

```

File: _pages/production.md [hunk 2]
```diff
@@ -2,81 +2,150 @@
...
-* OpenID Connect: <https://secure.login.gov/api/openid_connect/certs>
-* SAML: <https://secure.login.gov/api/saml/metadata{{ page.saml_year }}>
+-   [Signed Interagency Agreement (IAA) listing this integration ](https://docs.google.com/document/d/1JrseKerp-FF_uiGElew9z-ltT_l68dgmSFU7so8tQjE/edit#heading=h.gxf7gmdb96uf)
 
-## Deployments
+-   Authority To Operate (ATO) approved environment
 
-All changes to integrations between Login.gov and your application must be reviewed and deployed. **We ask for at least 2 weeks notice for new integrations and changes to existing integrations. Push Notification URLs may require 3 weeks notice in order to allow the domain for outbound communication.** Regular deployments occur every Thursday by the close of the business day. If the regular deployment occurs on a holiday, then it will be completed the following Monday.
+-   [New app on the dashboard ](https://docs.google.com/document/d/1JrseKerp-FF_uiGElew9z-ltT_l68dgmSFU7so8tQjE/edit#heading=h.rl9bxiu1u854)
```
Comment (zachmargolis) [inline _pages/production.md:38]: 1. there's an extra space here in this link
2. this links to a private (to us) doc, is the goal to link to a different heading in this same article?
3. 
```suggestion
-   [New app on the dashboard](https://docs.google.com/document/d/1JrseKerp-FF_uiGElew9z-ltT_l68dgmSFU7so8tQjE/edit#heading=h.rl9bxiu1u854)
```

File: _pages/production.md [hunk 3]
```diff
@@ -2,81 +2,150 @@
...
 
-Our integration documentation includes endpoint urls for our sandbox environment <https://idp.int.identitysandbox.gov/>. Our production environment is located at <https://secure.login.gov/>. The URL path to each endpoint remains the same. Only the domain will change. For example, the authorization endpoint will change as follows:
-* OpenID Connect: <https://secure.login.gov/openid_connect/authorize>
-* SAML: <https://secure.login.gov/api/saml/auth{{ page.saml_year }}>
+Make sure you have the following items ready before you start the deployment process:
 
-Please be aware that the IDP certificate (X509 Certificate) in the production environment is different from the IDP certificate in the sandbox environment. The production IDP certificates can be found here:
-* OpenID Connect: <https://secure.login.gov/api/openid_connect/certs>
-* SAML: <https://secure.login.gov/api/saml/metadata{{ page.saml_year }}>
+-   [Signed Interagency Agreement (IAA) listing this integration ](https://docs.google.com/document/d/1JrseKerp-FF_uiGElew9z-ltT_l68dgmSFU7so8tQjE/edit#heading=h.gxf7gmdb96uf)
```
Comment (zachmargolis) [inline _pages/production.md:34]: this is the same private doc again

**Review by karenstrother — COMMENTED** [review]
- Line 9 & 10 Production endpoint and href are duplicate of lines 15 & 16

File: _pages/production.md [hunk 4]
```diff
@@ -2,81 +2,150 @@
...
+If you encounter errors or have questions after completing these steps, please [submit a technical support ticket](https://zendesk.login.gov/).
+
+Production endpoints
+--------------------
+
+You will need to configure your application to point to the following endpoint: 
+
+-   OpenID Connect: [https://secure.login.gov/openid_connect/authorize](https://secure.login.gov/openid_connect/authorize)
+
+-   SAML: [https://secure.login.gov/api/saml/auth2022](https://secure.login.gov/api/saml/auth2022)
```
Comment (zachmargolis) [inline _pages/production.md:109]: I think our style has to been to use monospace fonts for URLs that are meant as configuration:
```suggestion
-   OpenID Connect: `https://secure.login.gov/openid_connect/authorize`

-   SAML: `https://secure.login.gov/api/saml/auth2022`
```

File: _pages/production.md [hunk 5]
```diff
@@ -2,81 +2,150 @@
...
+
+Our integration documentation includes endpoint urls for our sandbox environment https://idp.int.identitysandbox.gov/. Our production environment is located at https://secure.login.gov/. The URL path to each endpoint remains the same. Only the domain will change.
```
Comment (zachmargolis) [inline _pages/production.md:111]: based on the GDoc, looks like we want to monospace these URLs:
```suggestion
Our integration documentation includes endpoint urls for our sandbox environment `https://idp.int.identitysandbox.gov/`. Our production environment is located at `https://secure.login.gov/`. The URL path to each endpoint remains the same. Only the domain will change.  
```

**Review by karenstrother — COMMENTED** [review]
Line 25 - this should include the warning box formatting (<div class="usa-alert usa-alert--warning">
  <div class="usa-alert__body">
    <p class="usa-alert__text">Banner text</p>
  </div>
</div>)
Line 36 - extra space between dashboard and end bracket ]
Line 50 to 53 should be indented (nested) under line 48
Line 54 should be indented (nested) under line 53

File: _pages/production.md [hunk 6]
```diff
@@ -2,81 +2,152 @@
...
+
+-   A [Login.gov production account](https://secure.login.gov)
+
+**Please note: The following items are required to launch your app to production:**
+
+-   All production urls should have .gov, .mil, or a dedicated .com address and point to an ATO-ed environment.
+
+-   You must include a logo for your application. [Learn more about our logo guidelines](https://developers.login.gov/design-guidelines/#agency-logo-guidelines).
+
+-   If this is a SAML integration (Not OpenID Connect), then please ensure that:
```
Comment (NatashaJIbrahim) [inline _pages/production.md:52]: The three points under this need to be nested under it since they're related to SAML integrations.

**Review by karenstrother — COMMENTED** [review]
Line 60 should be nested in line with line 56 as part of the 'If this is a SAML integration' bullet points.  That is the only change I see.

**Review by karenstrother — COMMENTED** [review]
Looks great!

**Comment by NatashaJIbrahim** [comment]
@devjohnsonAMI is there work that's still needed on this? Is there anything blocking here?

**Review by NatashaJIbrahim — APPROVED** [review]

**Comment by devjohnsonAMI** [comment]
> @devjohnsonAMI is there work that's still needed on this? Is there anything blocking here?

No. Followed all of Karen suggestions. Had to review how to create some of the spacing and bullet points. Looks good now.

**Comment by NatashaJIbrahim** [comment]
Yay! thank you!

**Pull request merged**